wsat-simple/README.adoc

```diff
@@ -145,13 +204,14 @@ INFO  [stdout] (TaskWorker-2) [SERVICE] all participants voted 'prepared', so co
 INFO  [stdout] (TaskWorker-2) [SERVICE] commit called on backend resource.
 ----
 
-NOTE: You can ignore the warning message `ARJUNA043219: Could not save recovery state for non-serializable durable WS-AT participant restaurantServiceAT` that is printed in the server console. This quickstart does not implement the required recovery hooks in the interest of making it easy to follow. In a real world production application, you should provide the required recovery code. For more information, see http://narayana.io/docs/product.
```

Comment:
I do not know why this Note was added instead of set the MockRestaurantManager Serializable. "in the interest of making it easy to follow" I would prefer to remove the warning. Do you agree with me @mmusgrov ?